docs/tools/proposer-based-timestamps-runbook.md

```diff
@@ -4,47 +4,51 @@ order: 3
 
 # Proposer-Based Timestamps Runbook
 
-Version v0.36 of Tendermint added new constraints for the timestamps included in
-each block created by Tendermint. The new constraints mean that validators may
-fail to produce valid blocks or may issue `nil` `prevotes` for proposed blocks
+Version v1.0 of CometBFT added new constraints for the timestamps included in
+each block created by CometBFT. The new constraints mean that validators may
+fail to produce valid blocks or may issue `nil` prevotes for proposed blocks
 depending on the configuration of the validator's local clock.
 
 ## What is this document for?
 
 This document provides a set of actionable steps for application developers and
 node operators to diagnose and fix issues related to clock synchronization and
-configuration of the Proposer-Based Timestamps [SynchronyParams](https://github.com/tendermint/tendermint/blob/master/spec/core/data_structures.md#synchronyparams).
+configuration of the [`SynchronyParams`](https://github.com/cometbft/cometbft/blob/main/spec/core/data_structures.md#synchronyparams) consensus parameters.
```

Comment:
Similar comment: can we make this URL relative, so we don't have to specify a branch name?

Reply:
See previous comment.